components/Pagination.vue

```diff
@@ -7,33 +7,33 @@
 				</nuxt-link>
 			</li>
 			<li v-if="currentPageNum > 2">
-				<nuxt-link :to="firstLink" class="pagination-link" aria-label="Goto page 1">
+				<nuxt-link :to="firstLink" class="pagination-link" aria-label="Go to page 1">
```

Comment:
I also wonder about this `aria-label` use. I can see the the benefit of providing "Page" context for the numeric link text but I feel that this may be better-served by including visually-hidden text within the link itself (using the `screen-reader-only` class).